composer/workflows/dataflowtemplateoperator_create_dataset_and_table_test.py

```diff
@@ -0,0 +1,32 @@
+from google.cloud import bigquery
```

Comment:
Can you also add a copyright header?

Reply:
good catch

Reply:
Thanks for providing examples Takashi. Since I don't really code in Python, I'm not super familiar with standard libraries vs. other libraries. Is there an easy way to differentiate the libraries?

Reply:
@sofisl 

If you add something to `requirements.txt` or `requirements-test.txt`, it's a 3rd party library. If you can just import something, it's very likely a standard library. It's obvious for the local imports.